apps/core/templatetags/core_tags.py

```diff
@@ -25,3 +25,14 @@ def navigation(context):
 @register.filter
 def hreflang_url(value, arg):
     return value.replace(arg, "en-latest", 1)
+
+
+@register.inclusion_tag("components/footer.html", takes_context=True)
+def get_footer_content(context):
+    footer = ""
+    if FooterContent.objects.first() is not None:
+        footer = FooterContent.objects.first()
+
+    return {
+        "footer": footer,
+    }
```

Comment:
Drive-by comment: we could improve this by reducing the duplicated query. And it seems in the template, we don't really handle the case where the `FooterContent` object isn't available, so we might as well skip the `if`...?

```suggestion
    return {
        "footer": FooterContent.objects.first(),
    }
```

Reply:
Thanks @laymonage. I will update the PR